nni/algorithms/compression/pytorch/auto_compress/auto_compress_engine.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+import json_tricks
+from typing import Optional, Callable
+
+from torch.nn import Module
+from torch.optim import Optimizer
+
+import nni
+from nni.retiarii.utils import import_
```

Comment:
I'd rather maintain another copy of `import_` here than import from retiarii.

Importing from another component looks weird and un-self-contained.

Reply:
add copy